tutorials/xr/deploying_to_android.rst

```diff
@@ -78,28 +61,20 @@ Next change the name of the export preset for the device you're setting this up
 And enable **Use Gradle Build**.
 If you want to use one-click deploy (described below), ensure that **Runnable** is enabled.
 
-If the vendors plugins were installed correctly you should find entries for the different headsets, select the entry for meta:
+If the vendors plugins were installed correctly you should find entries for the
+different headsets under **XR Features**. Change the **XR Mode** to **OpenXR**, then
+select the entry for your headset if you see one. If you don't see one enable the
+Khronos plugin. It's the official Khronos loader, in due time all headsets should
```

Comment:
This isn't entirely true anymore. We've started adding entries for different Android platforms even though they all use the Khronos loader. This because while they use the same loader, they have platform dependent android feature flags and permissions.

There is a hope that in the future we will have an entry just for Khronos that will allow you to build an APK that can be deployed to different headsets but this will require the industry to unify these settings.

I believe only Meta and Lynx do not use the Khronos loader at this time, and we do have plans to change this for Meta as Meta does support the Khronos loader.
Lynx probably does too (this is a Qualcomm spaces device if I'm not mistaken) but as we have no Qualcomm spaces devices to test or verify, we're stuck.